src/Intl/Transliterator/data/ascii_by_languages.php

```diff
@@ -0,0 +1,2670 @@
+        "\xef\xbe\xa0" => ' ', // HALFWIDTH HANGUL FILLER
+    ),
+    // commonly used in Word documents
+    'msword' => array(
+        '©' => '(C)',
+        '®' => '(R)',
+        '℞' => 'Rx',
+        '‐' => '-',
+        '‑' => '-',
+        '‒' => '-',
+        '–' => '-',
+        '—' => '-',
+        '―' => '-',
+        '︱' => '-',
+        '︲' => '-',
+        '‖' => '||',
+        '⁄' => '/',
+        '⁅' => '[',
+        '⁆' => ']',
+        '⁎' => '*',
+        '、' => ',',
+        '。' => '.',
+        '〈' => '<',
+        '〉' => '>',
+        '《' => '<<',
+        '》' => '>>',
+        '〔' => '[',
+        '〕' => ']',
+        '〘' => '[',
+        '〙' => ']',
+        '〚' => '[',
+        '〛' => ']',
+        '︐' => ',',
+        '︑' => ',',
+        '︒' => '.',
+        '︓' => ':',
+        '︔' => ';',
+        '︕' => '!',
+        '︖' => '?',
+        '︙' => '...',
+        '︰' => '..',
+        '︵' => '(',
+        '︶' => ')',
+        '︷' => '{',
+        '︸' => '}',
+        '︹' => '[',
+        '︺' => ']',
+        '︽' => '<<',
+        '︾' => '>>',
+        '︿' => '<',
+        '﹀' => '>',
+        '﹇' => '[',
+        '﹈' => ']',
+        '×' => '*',
+        '÷' => '/',
+        '−' => '-',
+        '∕' => '/',
+        '∖' => '\\',
+        '∣' => '|',
+        '∥' => '||',
+        '≪' => '<<',
+        '≫' => '>>',
+        '⦅' => '((',
+        '⦆' => '))',
+        '〇' => '0',
+        '′' => '\'',
+        '〝' => '"',
+        '〞' => '"',
+        '«' => '<<',
+        '»' => '>>',
+        '‘' => "'",
+        '’' => "'",
+        '‚' => ',',
+        '‛' => "'",
+        '“' => '"',
+        '”' => '"',
+        '„' => '"',
+        '‟' => '"',
+        '‹' => '<',
+        '›' => '>',
+        '…' => '...',
+    ),
+    // Currency
+    //
+    // url: https://en.wikipedia.org/wiki/Currency_symbol
+    'currency_short' => array(
+        '€' => 'EUR',
+        '$' => '$',
+        '₢' => 'Cr',
+        '₣' => 'Fr.',
+        '£' => 'PS',
+        '₤' => 'L.',
+        'ℳ' => 'M',
+        '₥' => 'mil',
+        '₦' => 'N',
+        '₧' => 'Pts',
+        '₨' => 'Rs',
+        'රු' => 'LKR',
+        'ரூ' => 'LKR',
+        '௹' => 'Rs',
+        'रू' => 'NPR',
+        '₹' => 'Rs',
+        '૱' => 'Rs',
+        '₩' => 'W',
+        '₪' => 'NS',
+        '₸' => 'KZT',
+        '₫' => 'D',
+        '֏' => 'AMD',
+        '₭' => 'K',
+        '₺' => 'TL',
+        '₼' => 'AZN',
+        '₮' => 'T',
+        '₯' => 'Dr',
+        '₲' => 'PYG',
+        '₾' => 'GEL',
+        '₳' => 'ARA',
+        '₴' => 'UAH',
+        '₽' => 'RUB',
+        '₵' => 'GHS',
+        '₡' => 'CL',
+        '¢' => 'c',
+        '¥' => 'YEN',
+        '円' => 'JPY',
+        '৳' => 'BDT',
+        '元' => 'CNY',
+        '﷼' => 'SAR',
+        '៛' => 'KR',
+        '₠' => 'ECU',
+        '¤' => '$?',
+        '฿' => 'THB',
+        '؋' => 'AFN',
+    ),
+);
+
+$result =& $data;
+unset($data);
```

Comment:
why do we need to replace the variable here ?

Reply:
I just copy and past the code from the other polyfills. I think in PHP >= 7 we can return the array without this stuff? @nicolas-grekas